test/generators/html/odoc.css

```diff
@@ -0,0 +1,782 @@
+@charset "UTF-8";
```

Comment:
I think we should include these files as symlink, otherwise, they'd get out of date.

Reply:
ah good point yes! I'll fix that.